trymito.io/pages/data-app.tsx

```diff
@@ -0,0 +1,136 @@
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import Image from 'next/image';
+import Footer from '../components/Footer/Footer';
+import Header from '../components/Header/Header';
+import pageStyles from '../styles/Page.module.css';
+import textImageSplitStyles from '../styles/TextImageSplit.module.css'
+import titleStyles from '../styles/Title.module.css';
+import homeStyles from '../styles/Home.module.css';
+
+// Import Icons & Background Grid
+
+import { classNames } from '../utils/classNames';
+import StreamlitAppGallery, { DATA_VERIFICATION_STREAMLIT_APP_LINK } from '../components/StreamlitAppGallery/StreamlitAppGallery';
+import CTAButtons from '../components/CTAButtons/CTAButtons';
+import DashboardCTACard from '../components/CTACards/DashboardCTACard';
+
+const DataApp: NextPage = () => {
+
+  return (
+    <>
+      <Head>
+        <title>Python Spreadsheet in Streamlit App | Mito </title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="description" content="Mito is a streamlit spreadsheet component that enables no-code spreadsheet automation with Pandas code generation from within a streamlit app. | Mito" />
+      </Head>
+      
+      <Header/>
+    
+      <div className={pageStyles.container}>
+
+        <main className={pageStyles.main}>
+
+            <section className={classNames(titleStyles.title_card)}>
+                <h1 className={titleStyles.title}>
+                    Upgrade Python scripts to interactive Streamlit Apps
+                </h1>
+                <p className={titleStyles.description}>
+                    Turn one hour of automation savings into tens of hours by sharing automation scripts through Streamlit dashboards.
```

Comment:
This is kinda a pitch for streamlit, not really Mito in streamlit. Maybe that's fine.

Reply:
Yeah, I agree, but I think that's fine. Let's see how it goes